plugin/cmp.lua

```diff
@@ -47,65 +47,44 @@ end)
 
 misc.set(_G, { 'cmp', 'plugin', 'cmdline', 'leave' }, function()
   if vim.fn.expand('<afile>') ~= '=' then
-    vim.cmd [[
+    vim.cmd([[
       augroup cmp-cmdline
         autocmd!
       augroup END
-    ]]
+    ]])
     require('cmp.utils.autocmd').emit('CmdlineLeave')
   end
 end)
 
 misc.set(_G, { 'cmp', 'plugin', 'colorscheme' }, function()
-  highlight.inherit('CmpItemAbbrDefault', 'Pmenu', {
-    guibg = 'NONE',
-    ctermbg = 'NONE',
-  })
-  highlight.inherit('CmpItemAbbrDeprecatedDefault', 'Comment', {
-    gui = 'NONE',
-    guibg = 'NONE',
-    ctermbg = 'NONE',
-  })
-  highlight.inherit('CmpItemAbbrMatchDefault', 'Pmenu', {
-    gui = 'NONE',
-    guibg = 'NONE',
-    ctermbg = 'NONE',
-  })
-  highlight.inherit('CmpItemAbbrMatchFuzzyDefault', 'Pmenu', {
-    gui = 'NONE',
-    guibg = 'NONE',
-    ctermbg = 'NONE',
-  })
-  highlight.inherit('CmpItemKindDefault', 'Special', {
-    guibg = 'NONE',
-    ctermbg = 'NONE',
-  })
-  highlight.inherit('CmpItemMenuDefault', 'Pmenu', {
-    guibg = 'NONE',
-    ctermbg = 'NONE',
-  })
+  highlight.inherit('CmpItemAbbrDefault', 'Pmenu', { bg = 'NONE' })
+  highlight.inherit('CmpItemAbbrDeprecatedDefault', 'Comment', { bg = 'NONE' })
+  highlight.inherit('CmpItemAbbrMatchDefault', 'Pmenu', { bg = 'NONE' })
+  highlight.inherit('CmpItemAbbrMatchFuzzyDefault', 'Pmenu', { bg = 'NONE' })
+  highlight.inherit('CmpItemKindDefault', 'Special', { bg = 'NONE' })
+  highlight.inherit('CmpItemMenuDefault', 'Pmenu', { bg = 'NONE' })
   for name in pairs(types.lsp.CompletionItemKind) do
     if type(name) == 'string' then
-      vim.cmd(([[highlight default link CmpItemKind%sDefault CmpItemKind]]):format(name))
+      vim.api.nvim_set_hl(0, ('CmpItemKind%sDefault'):format(name), { link = 'CmpItemKind' })
     end
   end
 end)
 _G.cmp.plugin.colorscheme()
 
-vim.cmd [[
+vim.cmd([[
   highlight default link CmpItemAbbr CmpItemAbbrDefault
   highlight default link CmpItemAbbrDeprecated CmpItemAbbrDeprecatedDefault
   highlight default link CmpItemAbbrMatch CmpItemAbbrMatchDefault
   highlight default link CmpItemAbbrMatchFuzzy CmpItemAbbrMatchFuzzyDefault
   highlight default link CmpItemKind CmpItemKindDefault
   highlight default link CmpItemMenu CmpItemMenuDefault
-]]
```

Comment:
Thank you for your contribution!

But I don't know the reason why this code block doesn't touch on this PR. Could you explain it?

Reply:
That would be because I completely forgot about it, my bad! I will update this PR soon! 😂

Reply:
@hrsh7th I have updated my commit to resolve this issue